### PR TITLE
[improvement] paste media from clipboard

### PR DESCRIPTION
**Pull request by steveruizok**
This PR allows images and videos from the clipboard to be added to the current page.

### CODE REVIEW

File: packages/tldraw/src/state/TldrawApp.ts [hunk 1]
```diff
@@ -1854,11 +1871,16 @@ export class TldrawApp extends StateManager<TDSnapshot> {
           bindings: TDBinding[]
           assets: TDAsset[]
         } = JSON.parse(result)
+
         if (data.type === 'tldr/clipboard') {
           pasteInCurrentPage(data.shapes, data.bindings, data.assets)
         } else {
+          console.log(data.type)
```
Comment (Proful) [inline packages/tldraw/src/state/TldrawApp.ts:1878]: nit, pls remove console.log

File: packages/tldraw/src/hooks/useKeyboardShortcuts.tsx [hunk 2]
```diff
@@ -15,6 +15,27 @@ export function useKeyboardShortcuts(ref: React.RefObject<HTMLDivElement>) {
     [ref]
   )
 
+  React.useEffect(() => {
+    if (!app) return
+    const handlePaste = (e: ClipboardEvent) => {
+      let items = e.clipboardData?.items ?? []
```
Comment (Proful) [inline packages/tldraw/src/hooks/useKeyboardShortcuts.tsx:21]: It works so well. The solution is simple & standard. Not sure why I am not able to make it work when I tried long time back.

Thanks a lot for this PR :)

File: packages/tldraw/src/hooks/useKeyboardShortcuts.tsx [hunk 3]
```diff
@@ -15,6 +15,27 @@ export function useKeyboardShortcuts(ref: React.RefObject<HTMLDivElement>) {
...
+        var item = items[index]
+        if (item.kind === 'file') {
+          var file = item.getAsFile()
+          if (file) {
+            app.addMediaFromFile(file)
+          }
+        }
+      }
+    }
+
```
Comment (Proful) [inline packages/tldraw/src/hooks/useKeyboardShortcuts.tsx:32]: Works great! copy & paste multiple images. However as @judicaelandria commented in discord size can be scaled down.


- Figma looks like doesn't scale down the image (but tldraw default scaling looks bigger)
- When pasting multiple images, tldraw placing one image on top of other. However Figma places side by side (better UX)


image

Comment (judicaelandria) [reply packages/tldraw/src/hooks/useKeyboardShortcuts.tsx:32]: scaling down the image but preserving the aspect ratio

Comment (steveruizok) [reply packages/tldraw/src/hooks/useKeyboardShortcuts.tsx:32]: This would be a good separate issue!

**Pull request merged**